documentation/modules/oauth/con-oauth-reauthentication.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// assembly-oauth.adoc
+
+[id='con-oauth-reauthentication-{context}']
+= Session re-authentication mechanism
+
+The Kafka _SASL OAUTHBEARER_ mechanism supports a so called 're-authentication' mechanism.
+
+When enabled (by setting `maxSecondsWithoutReauthentication` attribute of 'oauth' type `authentication` listener configuration), the authenticated session expires when the access token expires.
+
```

Comment:
_Make line 12 part of the above paragraph because I think it's strongly related._

Reply:
AFAICT the line 12 ('The client must ...') is already part of the paragraph that starts with 'When the re-authentication mechanism ...'